### ISSUE TITLE
this._textProp errors for typing in contenteditable elements containing value attribute as number

### ISSUE THREAD

**Issue by hiteshdua1** [dwachss/bililiteRange#18]
Hi @dwachss ,

```function textProp(el) ```
returns the following error for ```li``` element which is contenteditable 
```
Uncaught TypeError: this._el[this._textProp].replace is not a function
    at W3CRange.length (VM1678 bililiteRange.js:143)
    at bililiteRange (VM1678 bililiteRange.js:68)
    at $.simulate.simulateKeySequence (VM1677 jquery.simulate.key-sequence.js:115)
    at new $.simulate (VM1675 jquery.simulate.js:45)
    at HTMLLIElement.<anonymous> (VM1675 jquery.simulate.js:34)
    at Function.each (VM1673 jquery.min.js:2)
    at init.each (VM1673 jquery.min.js:2)
    at init.$.fn.simulate (VM1675 jquery.simulate.js:33)
    at HTMLButtonElement.<anonymous> (VM1679:90)
    at HTMLButtonElement.dispatch (VM1673 jquery.min.js:3)
```

On further investigation i found that this is because of ```value``` attribute of ```li``` element, which returns a number 
so this condition was getting executed 
```if (typeof el.value != 'undefined') return 'value';``` in ```function textProp(el){``` method.

I believe we can put a ```typeof el.value ==="string"``` check to solve this.

fiddle to replicate the issue : http://jsfiddle.net/gauz8av8/6/

Please suggest , if i should create pull request for the same.

**Comment by dwachss** [comment]
Interesting. I didn't realized li's had values. Please create a pull request to correct this.  

    On Wednesday, July 5, 2017 5:49 AM, hiteshdua1 <notifications@github.com> wrote:
 

 Hi @dwachss ,function textProp(el)
returns the following error for li element which is contenteditable    at W3CRange.length (VM1678 bililiteRange.js:143)
    at bililiteRange (VM1678 bililiteRange.js:68)
    at $.simulate.simulateKeySequence (VM1677 jquery.simulate.key-sequence.js:115)
    at new $.simulate (VM1675 jquery.simulate.js:45)
    at HTMLLIElement.<anonymous> (VM1675 jquery.simulate.js:34)
    at Function.each (VM1673 jquery.min.js:2)
    at init.each (VM1673 jquery.min.js:2)
    at init.$.fn.simulate (VM1675 jquery.simulate.js:33)
    at HTMLButtonElement.<anonymous> (VM1679:90)
    at HTMLButtonElement.dispatch (VM1673 jquery.min.js:3)
On further investigation i found that this is because of value attribute of li element, which returns a number
so this condition was getting executed
if (typeof el.value != 'undefined') return 'value'; in function textProp(el){ method.I believe we can put a typeof el.value ==="string" check to solve this.fiddle to replicate the issue : http://jsfiddle.net/gauz8av8/6/Please suggest , if i should create pull request for the same.—

**Comment by dwachss** [comment]
closed via #19
